napari/_qt/qt_viewer_dock_widget.py

```diff
@@ -80,7 +97,94 @@ def keyPressEvent(self, event):
 
     def _set_title_orientation(self, area):
         if area in (Qt.LeftDockWidgetArea, Qt.RightDockWidgetArea):
-            features = self._initial_features
+            features = self._features
+            if features & self.DockWidgetVerticalTitleBar:
```

Comment:
syntactically can we use `and` and `or` instead of `&` and `^`? Curious what @jni and @kne42 think, but I find the explicit words easier to understand (particularly for new comers) and preferable

Reply:
no, those are [bitwise operators](https://docs.python.org/3/reference/expressions.html?highlight=bitwise#binary-bitwise-operations) and are syntactically distinct from `and` and `or` ... Qt uses them a lot when representing combinations of constants.

Reply:
ah ok great, thanks for the clarification - fine to leave as it